src/modules/Resources/Buckets.ts

```diff
@@ -26,159 +24,71 @@ class Buckets extends TagoIOModule<GenericModuleParams> {
+   * Get information about the Device.
+   * @param deviceID Device ID
+   *
+   * @deprecated Use the method from `Resources.devices` instead.
    */
-  public async info(bucketID: GenericID): Promise<BucketInfo> {
-    let result = await this.doRequest<BucketInfo>({
-      path: `/bucket/${bucketID}`,
-      method: "GET",
-    });
-    result = dateParser(result, ["created_at", "updated_at"]);
-
-    return result;
+  public async info(deviceID: GenericID) {
+    return await this.devices.info(deviceID);
   }
 
   /**
-   * Get Amount of data on the Bucket
-   * @param bucketID Bucket ID
+   * Get amount of data on the Device.
+   * @param deviceID Device ID
+   *
+   * @deprecated Use the method from `Resources.devices` instead.
    */
-  public async amount(bucketID: GenericID): Promise<number> {
-    const result = await this.doRequest<number>({
-      path: `/bucket/${bucketID}/data_amount`,
-      method: "GET",
-    });
-
-    return result;
+  public async amount(deviceID: GenericID) {
+    return await this.devices.amount(deviceID);
   }
 
   /**
-   * List variables inside the bucket
-   * @deprecated
-   * @default
-   * ```json
-   * optionsObj: {
-   *   showAmount: false
-   *   showDeleted: false
-   *   resolveOriginName: false
-   * }
-   * ```
-   * @param bucketID Bucket ID
-   * @param optionsObj Request options
-   */
-  public async listVariables(bucketID: GenericID, optionsObj?: ListVariablesOptions): Promise<VariablesInfo[]> {
-    const result = await this.doRequest<VariablesInfo[]>({
-      path: `/bucket/${bucketID}/variable`,
-      method: "GET",
-      params: {
-        amount: optionsObj?.showAmount || false,
-        deleted: optionsObj?.showDeleted || false,
-        resolveOriginName: optionsObj?.resolveOriginName || false,
-      },
-    });
-
-    return result;
-  }
-
-  /**
-   * Delete a bucket variable
-   * @param bucketID Bucket ID
-   * @param deleteParams Variable Details
-   */
-  public async deleteVariable(
-    bucketID: GenericID,
-    deleteParams: { variable: string; origin: string }
-  ): Promise<string> {
-    const result = await this.doRequest<string>({
-      path: `/bucket/${bucketID}/variable`,
-      method: "DELETE",
-      body: deleteParams || {},
-    });
-
-    return result;
-  }
-
-  /**
-   * Get all device associated with bucket
-   * @param bucketID Bucket ID
-   */
-  public async getDevicesAssociated(bucketID: GenericID): Promise<BucketDeviceInfo[]> {
-    const result = await this.doRequest<BucketDeviceInfo[]>({
-      path: `/bucket/${bucketID}/device`,
-      method: "GET",
-    });
-
-    return result;
-  }
-
-  /**
-   * @deprecated Use device.copyChunk or device.getData to build reports
    * Export Data from Bucket
    * @param buckets Array of JSON with get details
    * @param output Type of output
    * @param optionsObj Options of request
+   *
+   * @deprecated Use device.copyChunk or device.getData to build reports
    */
   public async exportData(
```

Comment:
Confirming if this should be removed as well (there's no alternative in `Devices`)